config/scribe.php

```diff
@@ -68,6 +68,12 @@
          */
         'use_value' => env('SCRIBE_AUTH_KEY'),
 
+        /*
+         * The value of the parameter to be displayed by Scribe in documentation.
+         * If this value is null, Scribe will use a random value.
+         */
+        'display_value' => null,
```

Comment:
Let's just make this default to something, so it's more obvious what it is.

Also, now that I think of it, `placeholder` would probably be more appropriate.

```suggestion
        'placeholder' => '{YOUR_AUTH_KEY}',
```